apps/docs/pages/guides/realtime/broadcast.mdx

```diff
@@ -298,25 +298,57 @@ This is currently available only in the Supabase JavaScript client version 2.37.
 
 You can also send a Broadcast message by making an HTTP request to Realtime servers. This is useful when you want to send messages from your server or client without having to first establish a WebSocket connection.
 
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="js"
+  queryGroup="language"
+>
+<TabPanel id="js" label="JavaScript">
 ```js
 const channel = client.channel('test-channel')
 
 // No need to subscribe to channel
 
 channel
-  .send({
-    type: 'broadcast',
-    event: 'test',
-    payload: { message: 'Hi' },
-  })
-  .then((resp) => {
-    console.log(resp)
-  })
+.send({
+type: 'broadcast',
+event: 'test',
+payload: { message: 'Hi' },
+})
+.then((resp) => {
+console.log(resp)
+})
```

Comment:
I think keeping the indents might have been nicer visually!

Reply:
`npm run format` keeps removing the indentation 😡